config/locales/en.yml

```
@@ -33,20 +33,20 @@ en:
   static_pages:
     about:
         beliefs:
-        - title: Education should be free and accessible
-          description: This curriculum itself is free and we tried to link to resources that are themselves free so anyone in the world can use them.
+        - title: Our lessons build upon each other
```

Nit: This one might resonate more if it highlighted the roadmap TOP provides. That is probably our biggest draw for users - getting them out of tutorial hell.

```suggestion
        - title: A full roadmap to becoming a developer
```